package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "^3.4.1"
+    "moment": "^2.29.4",
```

Comment:
I generally remove the `^` to stop dependencies automatically updating and potentially bringing breaking changes